pre_cache_action.sh

```diff
@@ -44,9 +44,19 @@ fi
 
 # Is length of string zero?
 if test -z "${packages}"; then
-  log "aborted"
-  log "Packages argument cannot be empty." >&2
-  exit 3
+  if [ "$IF_EMPTY_PACKAGES" = "ignore" ]; then
```

Comment:
This could just be a case statement or plain `if`s since it returns out of the block.

Reply:
Changed to a `case` statement.